src/main/kotlin/no/nav/helsearbeidsgiver/inntektsmelding/InntektsmeldingRepository.kt

```diff
@@ -81,6 +81,14 @@ class InntektsmeldingRepository(
                 }.map { it.toExposedInntektsmelding() }
         }
 
+    fun hent(navReferanseId: UUID): List<InntektsmeldingResponse> =
+        transaction(db) {
+            InntektsmeldingEntitet
+                .selectAll()
+                .where { InntektsmeldingEntitet.navReferanseId eq navReferanseId }
+                .map { it.toExposedInntektsmelding() }
+        }
```

Comment:
Pirk: `.map { it.toExposedInntektsmelding() }` kan gjøres utenfor `transaction`.

Reply:
Sikkert lurt forslag men når jeg forsøkte å gjøre dette så brakk testene. Så tror jeg lar det være for nå.